components/ReviewCard.test.js

```diff
@@ -247,6 +304,9 @@ describe('ReviewCard Component', () => {
     )
   })
   test('Should be able to add comment', async () => {
+    const updateCacheMock = jest.fn()
+    updateCache.mockImplementation(() => updateCacheMock())
```

Comment:
This has be done because UpdateCache tries to get the cache but there's no cache: (Error: `No Apollo Cache found)`

Reply:
I couldn't fix even after passing a new cache to the component. Any feedbacks are appreciated.

Reply:
not sure!